sustaingym/envs/building/stochastic_uncontrollable_generator.py

```diff
@@ -0,0 +1,281 @@
+import numpy as np
+import pandas as pd
+import scipy
+
+class StochasticUncontrollableGenerator():
+    def __init__(self, 
+                 building_env = None, 
+                 num_episodes: int = None):
+        """
+        Initializes a generator class for the uncontrollable features in BuildingEnv.
+
+        :param building_env: The instance of the BuildingEnv environment.
+        :param num_episodes: The number of episodes over which to collect observations
+            from the environment.
+        """
```

Comment:
Please make the docstring formatting consistent with Google-style Sphinx docstrings:
https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html

Generally speaking that means:


```python
def f(x: int | None = None) -> float:
    """One sentence description of f, ends in a period.

    Args:
        x: description of input x. If x is a numpy array, then this
            description must include the shape and dtype expected
            of x.

    Returns:
        y: squared value of x

    Raises:
        ValueError: if x is None
    """
    if x is None:
        raise ValueError("x should not be None")

    y = x**2
    return y
```